packages/genesys-spark-components/src/components/beta/gux-date-time/tests/__snapshots__/gux-date-time.spec.ts.snap

```diff
@@ -331,7 +331,7 @@ exports[`gux-date-time #render using non regional dates #render different dateti
 exports[`gux-date-time #render using non regional dates #render different datetime formats eg(short,medium,full,long) should render component as expected (2) 16`] = `
 <gux-date-time-beta datetime="2022-07-07T13:35:30.100Z" format="medium">
   <mock:shadow-root>
-    7. jul. 2022, 13:35:30
+    7. juli 2022, 13:35:30
```

Comment:
I think the failure is due to this not sure why it was picked up but will investigate it.